code/modules/power/supermatter/supermatter.dm

```diff
@@ -280,6 +280,7 @@
 	add_overlay(causality_field, TRUE)
 
 	var/speaking = "[emergency_alert] The supermatter has reached critical integrity failure. Emergency causality destabilization field has been activated."
+	SEND_SOUND(world, sound('sound/machines/engine_alert2.ogg'))
```

Comment:
Only issue here is `world` is going to include people on mining, and even in the lobby.